decode-morce.rb

```diff
@@ -0,0 +1,57 @@
+CODE = {
+  'A' => '.-',
+  'B' => '-...',
+  'C' => '-.-.',
+  'D' => '-..',
+  'E' => '.',
+  'F' => '..-.',
+  'G' => '--.',
+  'H' => '....',
+  'I' => '..',
+  'J' => '.---',
+  'K' => '-.-',
+  'L' => '.-..',
+  'M' => '--',
+  'N' => '-.',
+  'O' => '---',
+  'P' => '.--.',
+  'Q' => '--.-',
+  'R' => '.-.',
+  'S' => '...',
+  'T' => '-',
+  'U' => '..-',
+  'V' => '...-',
+  'W' => '.--',
+  'X' => '-..-',
+  'Y' => '-.--',
+  'Z' => '--..'
+}.freeze
+
+def decode_char(morce_code)
+   @character = CODE.key(morce_code)
+   @character
+end
+
+def decode_entire_word_helper(morce_code)
+  @characters = morce_code.split 
+  @join_character = ''
+  @characters.each do |n|
+    @join_character += decode_char(n)
+  end
+  @join_character
+end
+
+def decode_word(entire_word)
+  decode_entire_word_helper(entire_word)
+end
+
+def decode(entire_message)
+  @characterss = entire_message.split('   ')
+  @join_words = ' '
+  @characterss.each do |n|
+    @join_words += decode_entire_word_helper(n) + ' '
+  end
+  print @join_words
+end
+
+decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- [ ] Kindly implement the `decode_word` and `decode_char` so that the output will be as follows:
![image](https://user-images.githubusercontent.com/81932478/195298807-121c4228-4bf3-4a08-a20c-4a28ba91da52.png)

Your output
![image](https://user-images.githubusercontent.com/81932478/195298962-9be87767-2382-490e-8917-28dca293cf36.png)

Reply:
Well resolved 👍🏽